lib/swaps/Swaps.ts

```diff
@@ -1169,13 +1163,9 @@ class Swaps extends EventEmitter {
         break;
       case SwapPhase.PaymentReceived:
         assert(deal.phase === SwapPhase.SendingPayment, 'PaymentReceived can be only be set after SendingPayment');
-        this.logger.debug(`Payment received for deal with payment hash ${deal.rPreimage}`);
-        break;
-      case SwapPhase.SwapCompleted:
-        assert(deal.phase === SwapPhase.PaymentReceived, 'SwapCompleted can be only be set after PaymentReceived');
         deal.completeTime = Date.now();
         deal.state = SwapState.Completed;
-        this.logger.debug(`Swap completed. preimage = ${deal.rPreimage}`);
+        this.logger.debug(`Payment received for deal with payment hash ${deal.rPreimage}`);
```

Comment:
Should we log both the payment hash and the preimage here?

Reply:
Yes that makes sense, this is just an oversight.